nova/virt/vmwareapi/vm_util.py

```diff
@@ -1869,7 +1894,11 @@ def get_vm_detach_port_index(session, vm_ref, iface_id):
 
 
 def power_off_instance(session, instance, vm_ref=None):
-    """Power off the specified instance."""
```

Comment:
I am not sure that it is a good idea to change the interface.. I vaguely remember that other callers rely exactly on the opposite behaviour. I think, there is the power-off nova api call, and then instance termination, which both call this, and rely on the previous behaviour

Reply:
As far as I know, the callers were just calling this function without relying on the return value (since this method did not have any return statement before). Adding a return value shouldn't break any existing caller IMHO. So please correct me if i'm wrong, by posting the reference to the caller's code that rely on the previous behaviour.

Reply:
Yes, you are right.